MetroFramework/MetroFramework.csproj

```diff
@@ -254,43 +111,52 @@
     </None>
   </ItemGroup>
   <ItemGroup>
-    <BootstrapperPackage Include="Microsoft.Net.Client.3.5">
-      <Visible>False</Visible>
-      <ProductName>.NET Framework 3.5 SP1 Client Profile</ProductName>
-      <Install>false</Install>
-    </BootstrapperPackage>
-    <BootstrapperPackage Include="Microsoft.Net.Framework.3.5.SP1">
-      <Visible>False</Visible>
-      <ProductName>.NET Framework 3.5 SP1</ProductName>
-      <Install>true</Install>
-    </BootstrapperPackage>
-  </ItemGroup>
-  <ItemGroup>
-    <EmbeddedResource Include="Controls\MetroGrid.resx">
+    <EmbeddedResource Update="Controls\MetroGrid.resx">
       <DependentUpon>MetroGrid.cs</DependentUpon>
       <SubType>Designer</SubType>
     </EmbeddedResource>
-    <EmbeddedResource Include="Controls\MetroTextBox.resx">
+    <EmbeddedResource Update="Controls\MetroTextBox.resx">
       <DependentUpon>MetroTextBox.cs</DependentUpon>
     </EmbeddedResource>
     <EmbeddedResource Include="Localization\de\MetroToggle.xml" />
-    <EmbeddedResource Include="MessageBox\MetroMessageBoxControl.resx">
+    <EmbeddedResource Update="MessageBox\MetroMessageBoxControl.resx">
       <DependentUpon>MetroMessageBoxControl.cs</DependentUpon>
     </EmbeddedResource>
-    <EmbeddedResource Include="Properties\Resources.resx">
+    <EmbeddedResource Update="Properties\Resources.resx">
       <Generator>PublicResXFileCodeGenerator</Generator>
       <LastGenOutput>Resources.Designer.cs</LastGenOutput>
     </EmbeddedResource>
   </ItemGroup>
   <ItemGroup>
     <EmbeddedResource Include="Localization\en\MetroToggle.xml" />
   </ItemGroup>
-  <Import Project="$(MSBuildToolsPath)\Microsoft.CSharp.targets" />
-  <!-- To modify your build process, add your task inside one of the targets below and uncomment it. 
-       Other similar extension points exist, see Microsoft.Common.targets.
-  <Target Name="BeforeBuild">
-  </Target>
-  <Target Name="AfterBuild">
-  </Target>
-  -->
-</Project>
\ No newline at end of file
+  <ItemGroup>
+    <Compile Remove="Controls\MetroTilePart.cs" />
+    <Compile Remove="Forms\MetroMessageBox.cs" />
+  </ItemGroup>
+  <ItemGroup>
+    <PackageReference Include="System.Drawing.Common" Version="5.0.0-preview.6.20305.6" />
+    <PackageReference Include="System.Resources.Extensions" Version="5.0.0-preview.6.20305.6" />
+    <PackageReference Include="System.Windows.Extensions" Version="4.7.0" />
+  </ItemGroup>
+  <ItemGroup>
+    <Reference Include="PresentationCore">
+      <HintPath>..\..\..\..\..\..\Program Files\dotnet\packs\Microsoft.WindowsDesktop.App.Ref\5.0.0-preview.6.20308.1\ref\net5.0\PresentationCore.dll</HintPath>
```

Comment:
These HintPaths should be replaced by a `<UseWindowsForms>true</UseWindowsForms>`

Reply:
Unable to figure out how. Keyword seems unrecognised.
I updated the hintpaths though